aopy/postproc/base.py

```diff
@@ -149,6 +149,55 @@ def get_trial_targets(trials, targets):
         trial_targets[trial].append(targets[idx])
     return trial_targets
 
+def get_minimum_trials_per_target(target_idx, cond_mask):
+    '''
+    Get the minimum number of trials per target after restricting trials
+    
+    Args:
+        target_index (ntr): target index
+        cond_mask (ntr): boolean array to remove trials
+        
```

Comment:
I think this should be an optional argument. Also it should describe if 'True' keeps a trial or not.